README.md

```diff
@@ -1,6 +1,43 @@
-# Noto | Minimal Note-Taking App
-
-<img src="images/Icon.svg" alt="icon"/>
+<h1 align="center">
+  <img src="images/Icon.svg" alt="icon"/>
+  <br>
+  <b>Noto | Minimal Note-Taking App</b>
+</h1>
+
+<div align="center">
+  <a>
+    <img src="https://img.shields.io/badge/Android-3DDC84?style=for-the-badge&logo=android&logoColor=white" alt="Android">
+  </a>
+  <a>
+    <img src="https://img.shields.io/badge/Kotlin-purple?style=for-the-badge&logo=kotlin&logoColor=white" alt="Kotlin">
+  </a>
+  <a href="https://www.patreon.com/alialbaali">
+    <img src="https://img.shields.io/badge/Patreon-F96854?style=for-the-badge&logo=patreon&logoColor=white">
+  </a>
+  <a href="/LICENSE.md">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-green?style=for-the-badge&logo=apache" alt="License">
+  </a>
+</div>
+
```

Comment:
If you'd like, you can add more badges, which can be found here in these links:
https://github.com/alexandresanlim/Badges4-README.md-Profile
https://dev.to/envoy_/150-badges-for-github-pnk
https://github.com/Ileriayo/markdown-badges

I personally like to add these also:
![Kotlin](https://img.shields.io/badge/kotlin-%237F52FF.svg?style=for-the-badge&logo=kotlin&logoColor=white)
![Buy me a coffee](https://img.shields.io/badge/Buy_Me_A_Coffee-FFDD00?style=for-the-badge&logo=buy-me-a-coffee&logoColor=black)

I don't want to add too many as it might feel cluttered.